tests/dotnet/chart/templates/deployment.yaml

```diff
@@ -12,21 +12,20 @@ spec:
     metadata:
       labels:
         app: test-app-dotnet
-      annotations:
-        instrumentation.newrelic.com/inject-dotnet: "true"
+        app.newrelic.instrumentation: newrelic-dotnet-agent
     spec:
       containers:
         - name: test-app-dotnet
           image: e2e/test-app-dotnet:e2e
           imagePullPolicy: Never
           ports:
-            - containerPort: 80
+            - containerPort: {{ .Values.exposedPort }}
           env:
           - name: NEW_RELIC_APP_NAME
             value: k8s-e2e-test-app-dotnet
-          # used by the e2e Github action
+          # labels used by the e2e Github action
           - name: NEW_RELIC_LABELS
-            value: 'testKey:{{ .Values.scenarioTag | default "NOTSET" }}'
+            value: "testKey:{{ .Values.scenarioTag | default "NOTSET" }}"
```

Comment:
I don't think this will work, since we want `"NOTSET"` to be quoted

Reply:
The double brackets {{ }} are preprocessed by helm, and the quoting on NOTSET is consumed there. The final output from helm templating is this, which I ran by hand.

```
> helm template -s templates/deployment.yaml ./tests/python/chart
...
          - name: NEW_RELIC_LABELS
            value: "testKey:NOTSET"
...
```

Reply:
Ah! Ok, got it